src/custom.css

```diff
@@ -14,16 +14,17 @@
 }
 
 .btc-search .sui-layout-sidebar{
-    width: 30%;
+    overflow-x: hidden;
```

Comment:
I'm not really a fan of this approach. I'd prefer I think to use break points at 800px to make sure it uses the full screen on mobile